config/README.md

```diff
@@ -1,16 +1,72 @@
+The full model than additionally includes variables of interest, so fore example: `full: ~variable_of_interest + batch_effect_1 + batch_effect_2`.
+
+#### sleuth effect sizes
+
+Effect size estimates are calculated as so-called beta-values by `sleuth`.
+For binary comparisons (your variable of interest has two factor levels), they resemble a log2 fold change.
+To know which variable of interest to use for the effect size calculation, you need to provide its column name as the `primary_variable:`.
+And for sleuth to know what level of that variable of interest to use as the base level, specify the respective entry as the `base_level:`.
+
+### Lexogen 3' QuantSeq data analysis
+
+For Lexogen 3' QuantSeq data analysis, please set `experiment: 3-prime-rna-seq: activate: true` in the `config/config.yaml` file.
+For more information information on Lexogen QuantSeq 3' sequencing, see: https://www.lexogen.com/quantseq-3mrna-sequencing/
+In addition, for Lexogen 3' FWD QuantSeq data, we recommend setting the `params: cutadapt-se:` with:
+```
+    adapters: "-a 'r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=7;max_error_rate=0.005'"
+    extra: "--minimum-length 33 --nextseq-trim=20 --poly-a"
+```
+This is an adaptation of the [Lexogen read preprocessing recommendations for 3' FWD QuantSeq data](https://faqs.lexogen.com/faq/what-is-the-adapter-sequence-i-need-to-use-for-t-1).
+Changes to the recommendations are motivated as follows:
+* `-m`: We switched to the easier to read `--minimum-length` and apply this minimum length globally. In addition, we increase the minimum length to a default of 33 that makes more sense for kallisto quantification.
+* `-O`: Instead of this option, minimum overlap is specified per expression.
+* `-a "polyA=A{20}"`: We replace this with `cutudapt`s dedicated option for `--poly-a` tail removal (which is run after adapter trimming).
+* `-a "QUALITY=G{20}"`: We replace this with `cutudapt`s dedicated option for the removal artifactual trailing `G`s in NextSeq and NovaSeq data: `--nextseq-trim=20`.
+* `-n`: With the dedicated `cutadapt` options getting applied in the right order, this option is not needed any more.
+* `-a "r1adapter=A{18}AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000"`: We remove A{18}, as this is handled by `--poly-a`. We increase `min_overlap` to 7 and set the `max_error_rate` to the Illumina error rate of about 0.005, both to avoid spurious adapter matches being removed.
+* `-g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20"`: This is not needed any more, as `-a` option will lead to complete removal of read sequence if adapter is found at the start of the read, see: https://cutadapt.readthedocs.io/en/stable/guide.html#rightmost
+* `--discard-trimmed`: We omit this, as the `-a` with the adapter sequence will lead to complete read sequence removal if adapter is found at start, and the `--minimum-length` will then discard such empty reads.
```

Comment:
@manuelphilip This should explain all the details of the changes to the cutadapt command(s) from the original documentation that I made. As the cutadapt manual states, it will execute the respective trimming actions in the correct order, see here:
https://cutadapt.readthedocs.io/en/stable/reference.html#read-modification-order

Namely, it will do them in the following order:
1. Do quality trimming. `--nextseq-trim` is effectively a special version of quality trimming, so this should run first. And this makes the most sense, as the trailing `G`s that NextSeq data might have, will be at the very end of reads.
2. Trim off adapters specified with `-a`. These should be the outermost after quality trimming.
3. `--poly-a` tail removal should come after quality and adapter trimming, as the poly-A tails are part of the actual target sequence / fragments to be sequenced.

I double-checked with some minimal QuantSeq data from the original publication that you provided, but feel free to point any of your actual data analyses to this branch and try it out. Results are not exactly identical, but roughly identical. And I expect the new version provided here to actually be better, as both the `--nextseq-trim` and `--poly-a` flags are dedicated flags for the tasks that are supposed to be better than highjacking `-a` and `-g` for this, and they enforce the right order of trimming.

Reply:
Aha..This sound perfect. Thank you @dlaehnemann for investigating your time into cutadapt section. I will also check it with the inhouse reads, but I think it should give us similar results.